src/libse/Common/Settings.cs

```diff
@@ -1323,6 +1314,17 @@ public GeneralSettings()
             DialogStyle = DialogType.DashBothLinesWithSpace;
             ContinuationStyle = ContinuationStyle.None;
 
+            if (Configuration.IsRunningOnLinux)
+            {
+                SubtitleFontName = Configuration.DefaultLinuxFontName;
+                VideoPlayerPreviewFontName = Configuration.DefaultLinuxFontName;
+            }
+            else if (Environment.OSVersion.Version.Major < 6 || Environment.OSVersion.Version.Major == 6 && Environment.OSVersion.Version.Minor < 2) // Vista/Win2008Server/Win7
```

Comment:
As you can see here https://stackoverflow.com/a/2819962/14349868
Windows 7 is 6, but the minor is 1, so it should be like this, I think.

Reply:
I think you can remove "else stuff"" as  WinXP is no longer supported.
"Times New Roman" was one of the few fonts on WinXP that had some Unicode support.

Reply:
So Windows 7 has "Tahoma"? The comment mentions Windows 7 as well.

Side note: I moved from XP to 10, never worked with 7. ^^"

Reply:
Okay, given that the code was using Tahoma on Win7 and no one complained, I guess it's safe to say that it works on Win7.

Reply:
Yes, Unicode support in Win7 is fine :)